keycloak/keycloak-ui
# Replaced Tabs to RoutableTabs implementation in realm-settings - #3981

## Motivation
Closes #3981

All subtabs in realm settings are now implemented in the same way and are using RoutableTabs instead of Tabs component. 

## Verification Steps
1. Go to `Realm setttings` and navigate through tabs and subtabs.
2. Verify that the subtabs have now left padding.

## Checklist:
